### DISCUSSION TITLE
Claude系列的模型会支持插件系统吗？

### DISCUSSION THREAD

**Discussion by Blindinlights** [lobehub/lobehub#1937] [General Question]
现在claude也有tool_use接口,也就是function_call, 不知道有没有计划让插件支持claude?
[anthropic官方文档](https://docs.anthropic.com/claude/docs/tool-use)

**Comment by arvinxx** [comment]
会，这个月下半月搞

**Comment by BrandonStudio** [comment]
#1472 #1929 #1940
